### ISSUE TITLE
create handler to write logs to a file (w/setting in options dialog) (Trac #24)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#28]
Create handler to write logs to a file and make the corresponding options dialog.

Migrated from http://microfluidics.utoronto.ca/ticket/24
```json
{
    "status": "closed", 
    "changetime": "2015-01-06T12:57:37", 
    "description": "Create handler to write logs to a file and make the corresponding options dialog.", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1420549057189277", 
    "component": "microdrop", 
    "summary": "create handler to write logs to a file (w/setting in options dialog)", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:26:10", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T21:13:12: 
* christian@fobel.net commented:

> commit 4f52176fb9dcc3ba12d20202d316826a8abd295a
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Jan 4 15:25:06 2012 -0500
> 
>     Add logging file handler to options dialog...
>     
>     Add logging file handler to options dialog. Fixes #28.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20120104T21:13:31: cfobel changed type from "defect" to "enhancement"

**Comment by cfobel** [comment]
Trac update at 20140314T05:59:17: 
* ryan commented:

> Logging to a file functionality seems to be broken.


* ryan changed resolution from "fixed" to ""
* ryan changed status from "closed" to "reopened"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "core" to "microdrop"

**Comment by cfobel** [comment]
Trac update at 20150105T20:48:00: 
* cfobel changed resolution from "" to "fixed"
* cfobel changed status from "reopened" to "closed"

**Comment by cfobel** [comment]
Trac update at 20150106T12:57:37: Christian Fobel <christian@fobel.net> commented:

> In changeset 4d36ef49316e9ad71c60a755fcd27c6a9a8e7d97:
> ```
> #!CommitTicketReference repository="microdrop" revision="4d36ef49316e9ad71c60a755fcd27c6a9a8e7d97"
> Fix #28, logging to file
> 
> Prior to this commit, logging to file was not working since the log-file
> settings were incorrectly being loaded from the
> `'microdrop.gui.main_window_controller'` app options.  This commit corrects the
> mistake and now loads the log file settings from the app options of the
> `'microdrop.app'` plugin.
> ```
